test/files/neg/badimport.check

```diff
@@ -0,0 +1,4 @@
+badimport.scala:2: error: missing import selector collection._
+import collection
+       ^
```

Comment:
I think I'd prefer to keep the existing ". expected" message, and just work to improving its position.

This position is an better than the status quo, but I think the best spot would be at the end of "collection".

 but tweaked to locate it at the end of the previous token, rather than at the start of the next one.

Reply:
Sure, that's helpful.